gpu/kinfu/tools/CMakeLists.txt

```diff
@@ -8,8 +8,10 @@ if(NOT VTK_FOUND)
 else()
   set(DEFAULT TRUE)
   set(REASON)
-  set(VTK_USE_FILE "${VTK_USE_FILE}" CACHE INTERNAL "VTK_USE_FILE")
-  include("${VTK_USE_FILE}")
+  if(${VTK_VERSION} VERSION_LESS 9.0)
+    set(VTK_USE_FILE ${VTK_USE_FILE} CACHE INTERNAL "VTK_USE_FILE")
```

Comment:
Is this needed here? There's already a `VTK_USE_FILE`, so why store it in `VTK_USE_FILE`?